dependencies/pom.xml

```diff
@@ -1152,20 +1130,19 @@
                 <version>${version.lib.activemq}</version>
             </dependency>
             <dependency>
-                <groupId>io.micronaut</groupId>
+                <groupId>io.micronaut.validation</groupId>
```

Comment:
Examples will need to be updated, see [examples/integrations/micronaut/data/pom.xml](https://github.com/helidon-io/helidon-examples/blob/b2d18a389efb9372f9b3cb47d27d5c5efea9d982/examples/integrations/micronaut/data/pom.xml#L127)

Reply:
The pipeline builds the dev-4.x branch, we only need to merge your example PR after this PR is merged.